src/Plane.js

```diff
@@ -0,0 +1,16 @@
+class Plane {
+
+    constructor(id) {
+        if (id === undefined) {
+            throw new Error(`Plane ID does not exist`);
+        } else {
+            this.id = id;
+        }
+    }
+
```

Comment:
Class Plane constructor: Potentially should have added a condition that the ID had to be a string or throws an error, or alternatively changed anything that was passed into the ID into a string using a toString() method.